homework/Classes_scope_ and_namespaces/task3.py

```diff
@@ -0,0 +1,63 @@
+
+
+CHANNELS = ["BBC", "Discovery", "TV1000"]
+
+
+class TVController:
+
+    current_channel = CHANNELS[0]   # default channel
+
+    def __init__(self, list_of_channels):
+        self.list_of_channels = list_of_channels
+
+    def first_channel(self):
+        TVController.current_channel = CHANNELS[0]
```

Comment:
Similar to the previous suggestion, use `self` instead of `TVController` here and below. With the current implementation if you create multiple instances of `TVController` later on and switch channels for any of them - that would switch channels for all of them at the same time (which doesn't seem right for me).